packages/genesys-spark-components/src/components/stable/gux-popup/tests/__snapshots__/gux-popup.spec.ts.snap

```diff
@@ -5,7 +5,7 @@ exports[`gux-popup #render should render component as expected (1) 1`] = `
   <template shadowrootmode="open">
     <div aria-disabled="false" class="gux-target-container">
       <slot name="target"></slot>
-      <div class="gux-popup-container">
+      <div class="gux-popup-container gux-sr-only-clip" style="width: 0px; overflow: hidden; left: NaNpx; top: 2px;">
```

Comment:
I don't like that left is `NaN` px

Reply:
If you want to address this if is set [here](https://github.com/MyPureCloud/genesys-spark/pull/659/files#diff-5baea0a22dcd3dd2e2d444a42cf7a93835be1222d84c6ccaf4314064759b129aR130). You can probably put in some guards to not set it is it is not a number.